pkg/expapi/types.go

```diff
@@ -293,65 +293,65 @@ type DeploymentList struct {
 	Items []Deployment `json:"items"`
 }
 
-// DaemonSpec is the specification of a daemon.
-type DaemonSpec struct {
-	// Selector is a label query over pods that are managed by the daemon.
+// DaemonSetSpec is the specification of a daemon.
+type DaemonSetSpec struct {
+	// Selector is a label query over pods that are managed by the daemon set.
 	// Must match in order to be controlled.
 	// If empty, defaulted to labels on Pod template.
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/labels.md#label-selectors
 	Selector map[string]string `json:"selector,omitempty"`
 
 	// Template is the object that describes the pod that will be created.
-	// The Daemon will create exactly one copy of this pod on every node
+	// The DaemonSet will create exactly one copy of this pod on every node
 	// that matches the template's node selector (or on every node if no node
 	// selector is specified).
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#pod-template
 	Template *api.PodTemplateSpec `json:"template,omitempty"`
 }
 
-// DaemonStatus represents the current status of a daemon.
-type DaemonStatus struct {
-	// CurrentNumberScheduled is the number of nodes that are running exactly 1 copy of the
-	// daemon and are supposed to run the daemon.
+// DaemonSetStatus represents the current status of a daemon set.
+type DaemonSetStatus struct {
+	// CurrentNumberScheduled is the number of nodes that are running exactly 1
+	// daemon pod and are supposed to run the daemon.
```

Comment:
s/daemon/daemon pod/ (though I think this is not really necessary since now "daemon" no longer means "daemon set")